### DISCUSSION TITLE
Hypervisor on M1 iPad No Jailbreak or TrollStore

### DISCUSSION THREAD

**Discussion by stossy11** [utmapp/UTM#4737] [UTM Help (iOS)]
So i Just got an M1 iPad Air and i was wondering if now i can use hypervisor or maybe use Virtualization setting on UTM
and i have No jailbreak because i am on iPadOS 16 and TrollStore Doesnt work

i have an 
iPad Air 5 
8GB Of RAM
M1 Chip

**Comment by TheToto** [comment]
You can’t currently on iPadOS 16
